streams/src/main/java/org/apache/kafka/streams/state/StreamsMetadata.java

```diff
@@ -82,21 +91,20 @@ public boolean equals(final Object o) {
             return false;
         }
         final StreamsMetadata that = (StreamsMetadata) o;
-        if (!hostInfo.equals(that.hostInfo)) {
-            return false;
-        }
-        if (!stateStoreNames.equals(that.stateStoreNames)) {
-            return false;
-        }
-        return topicPartitions.equals(that.topicPartitions);
+        if (!hostInfo.equals(that.hostInfo)) return false;
```

Comment:
nit: newline for better IDE debugging.

Reply:
Also I think the checkstyle will fail as well for this code style.

Reply:
Thanks, will edit this.

Reply:
Please add `{}`, too